packages/oidc-js/src/utils/sign-in.ts

```diff
@@ -694,6 +701,20 @@ export const getUserInfo = (config: ConfigInterface | WebWorkerConfigInterface):
     };
 };
 
+/**
+ * This function decodes the payload of an id token and returns it.
+ *
+ * @param {string} idToken - The id token to be decoded.
+ *
+ * @return {DecodedIdTokenPayloadInterface} - The decoded payload of teh id token.
+ */
+const decodeIDToken = (idToken: string): DecodedIdTokenPayloadInterface => {
+    const words = base64URLDecode.parse(idToken.split(".")[ 1 ]);
+    const utf8String = utf8.stringify(words);
+    const payload = JSON.parse(utf8String);
```

Comment:
Would be better to add a try catch here.

Reply:
Fixed by 3aaab96